src/app/static/src/app/components/genericChart/Plotly.vue

```diff
@@ -87,27 +87,26 @@ import { StyleValue } from "vue";
             }
         },
         methods: {
-            drawChart: function() {
+            drawChart: async function() {
                 this.rendering = true;
-                setTimeout(() => {
-                    try {
-                        const el = this.$refs.chart;
-                        const drawFunc = this.layoutRequired ? Plotly.newPlot : Plotly.react;
-                        this.layoutRequired = false;
-                        drawFunc(el as HTMLElement, this.data.data as any, this.data.layout, this.data.config as any);
-                    }
-                    finally {
-                        this.rendering = false;
-                    }
-                }, 0);
+                const el = this.$refs.chart;
+                const drawFunc = Plotly.react;
+                this.layoutRequired = false;
+                const dataScatterGL = this.data.data.map((data: any) => {return {...data, type: "scattergl"}})
+                console.log(dataScatterGL)
+                await drawFunc(el as HTMLElement, dataScatterGL, this.data.layout, {...this.data.config as any});
+                this.rendering = false;
```

Comment:
this was my initial attempt at fixing the plotly graph, ignore this, this gets a PR that will be up shortly, all these changes are going to be discarded